Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.yml

```diff
@@ -28,14 +33,20 @@ comment: |-
 commonfields:
   id: SendEmailReply
   version: -1
-enabled: false
+enabled: true
 name: SendEmailReply
-script: '-'
+script: ''
 subtype: python3
-system: false
-timeout: '0'
+system: true
 type: python
 fromversion: 5.0.0
-dockerimage: demisto/python3:3.9.8.24399
+dockerimage: demisto/python3:3.10.1.26972
 tests:
 - No tests (auto formatted)
+contentitemexportablefields:
+  contentitemfields:
+    fromServerVersion: ""
+tags: []
+scripttarget: 0
+runonce: false
+runas: DBotWeakRole
```

Comment:
This is the weak role, which is ok, but I see use of `demisto.executeCommand('executeCommandAt')` and `demisto.executeCommand('setIncident')` (which by the way should use execute_command instead) in this pr so it might fail on permissions.. Are we sure it is working?

Reply:
I have replaced `executeCommandAt` in this script with `appendContext`.  I had copied this function from PreprocessEmail.py where `executeCommandAt` is indeed needed to set context on another existing incident.

Even with `executeCommandAt` this was working fine with DBotWeakRole, but after my change it won't be an issue either way.